.github/stale.yml

```diff
@@ -0,0 +1,42 @@
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 3
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 1
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - bug
+  - Bug(Potintail)
```

Comment:
There is a typo here I can't edit the label since I don't have permissions ;)

Reply:
@w7shdev permission given 😁

Reply:
Thanks & done